### DISCUSSION TITLE
Hot end fan noisy when unheated - disable below 30C?

### DISCUSSION THREAD

**Discussion by Gamma36** [Jyers/Marlin#1441] [Ideas]
The NOISY hot end fan runs continuously, even if unheated. The schematics indicate it could be PWM controlled.

So logically it should be off below a certain threshold, say 30C, and maybe half speed below 50C?

When setting up and creating Z-meshes there is no need for the fan, if there is no heating. Seems like a bug to me.

**Comment by Starker3** [comment]
Only the part cooling fan can be PWM controlled and it is off by default. The hotend cooling fan on the E3V2 is connected directly to the power supply so its not possible to turn this fan on/off through firmware

**Reply by Gamma36** [reply]
Thanks for the info.

I'm an electronics engineer and looking at the schematics the hot end fan could be turned off if the chassis fan is off (and delayed for a few minute if required). So at power up and levelling all fans would be silent.

Another option is a temperature sensor on the hot end fan, slow it below 50 and off at 30

If there is a spare output pin we could use that.

**Reply by Starker3** [reply]
I haven't looked at the schematics, but AFAIK in the default setup the hotend fan is wired directly to the PSU. If you rewired it to use a spare pin you could probably figure out a way to turn it off while the printer is idle. If you can get it working in the firmware then you can use the sensor for the hotend to detect when the temp is low enough to turn it off.

**Reply by Gamma36** [reply]
Awesome. Great minds think alight ha ha.

Using a FET is ideal, could even vary the speed if the spare pin can connect to a PWM register.

As for hijacking a spare pin, is there a consensus about a standard use for spare pins?
